README.md

```diff
@@ -15,6 +15,8 @@ React Shared Components Library System for Forcepoint. Built on Vite.
 
 ## Library reference
 
+This section provides a concise overview of the key classes and functions available in our library, aiding developers in understanding and utilizing the provided functionalities effectively.
+
 ### Include the Tailwind preset
```

Comment:
What do you think of including this line after line 58?

Reply:
We can leave it as it is for now, we can improve the documentation in the future